ipaserver/install/plugins/adtrust.py

```diff
@@ -325,6 +321,28 @@ def execute(self, **options):
         return False, ()
 
 
+def get_gidNumber(ldap, env):
+    # Read the gidnumber of the fallback group
+    dn = DN(('cn', ADTRUSTInstance.FALLBACK_GROUP_NAME),
+            env.container_group,
+            env.basedn)
+
+    try:
+        entry = ldap.get_entry(dn, ['gidnumber'])
+        gidNumber = entry.get('gidnumber')
```

Comment:
``LDAPEntry.get()`` always returns a list. You must use ``entry.single_value.get('gidNumber')`` if you want the GID number.

```
>>> entry = conn.get_entry(DN(('cn', 'editors'), api.env.container_group, api.env.basedn))
>>> entry
LDAPEntry(ipapython.dn.DN('cn=editors,cn=groups,cn=accounts,dc=ipa,dc=example'), {'objectClass': [b'top', b'groupofnames', b'posixgroup', b'ipausergroup', b'ipaobject', b'nestedGroup'], 'gidNumber': [b'487600002'], 'description': [b'Limited admins who can edit other users'], 'cn': [b'editors'], 'ipaUniqueID': [b'f455854e-f88b-11e8-99ba-001a4a231543']})
>>> entry.get('gidNumber')
['487600002']
>>> entry.single_value.get('gidNumber')
'487600002'
```

Reply:
Actually, I will not change this place because the 'gidNumber' returned from `get_gidNumber` is directly assigned to an attribute on other `LDAPEntry` objects. This change means a need to do `[]` wrapping in other places for nothing because no other operation is done with this value.